src/views/catalog/customize/constants.ts

```diff
@@ -1,2 +1,15 @@
 export const DEFAULT_NAMESPACE = 'default';
 export const NAME_INPUT_FIELD = 'NAME';
+
+export const WINDOWS_DRIVERS_DISK = 'windows-drivers-disk';
+export const VIRTIO_WIN_IMAGE = 'virtio-win-image';
+
+export const VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAMES = ['v2v-vmware', 'virtio-win'];
```

Comment:
'v2v-vmware' is depracted in 4.9, and removed in 4.10 we don't support it in 4.11